tuf/repository_tool.py

```diff
@@ -2167,8 +2170,9 @@ def delegate(self, rolename, public_keys, list_of_targets, threshold=1,
         file specified by 'target/other_role'.
 
       restricted_paths:
-        A list of restricted directory or file paths of 'rolename'.  Any target
-        files added to 'rolename' must fall under one of the restricted paths.
+        A list of delegated paths, or glob patterns, of 'rolename'.  Any target
+        files added to 'rolename' must match one of the restricted glob
+        patterns.
```

Comment:
Can `restricted_paths` jn other functions be glob patterns too? I think it should be clear when it must be a path and when it can be a glob pattern. E.g. in `add_restricted_paths` it says it raises an error of `restricted_paths` is not a string path.